creator-node/src/ffmpeg.js

```diff
@@ -26,9 +26,10 @@ function segmentFile (fileDir, fileName, { logContext }) {
       '-hls_segment_filename', path.resolve(fileDir, 'segments', 'segment%05d.ts'),
       // "-vn" flag required to allow track uploading with album art
       // https://stackoverflow.com/questions/20193065/how-to-remove-id3-audio-tag-image-or-metadata-from-mp3-with-ffmpeg
-      '-vn',
+      '-vn', // skip inclusion of video
```

Comment:
hm.. this comment conflicts with the comment on line 27. could we double check this?

Reply:
From the docs:

> The -vn / -an / -sn / -dn options can be used to skip inclusion of video, audio, subtitle and data streams respectively, whether manually mapped or automatically selected, except for those streams which are outputs of complex filtergraphs.

Source: https://ffmpeg.org/ffmpeg.html#toc-Stream-selection

It was hard for me to really understand what was happening between the comment and the `-vn` flag, which is why I included the new comment. Essentially, if someone uploads an mp3 with album art without the `-vn` flag, we treat it as a video file and not an audio file. With `-vn` we ignore the album art at transcode time which allows the user to upload the mp3 with album art to our system since only the audio is getting processed.

Reply:
ohhh interesting! didn't know that 😄  thanks for the update! this description clarifies things so much. could we incorporate ^ comment into the PR?

Reply:
For sure! How does the current version look?

Reply:
looks good!